scss/components/_card.scss

```diff
@@ -67,7 +67,7 @@ $card-margin: $global-margin !default;
   box-shadow: $shadow;
 
   overflow: hidden;
-  color: $card-font-color;
+  color: $color;
```

Comment:
if, before my suggested change, you write

`.mycard {
   @include card-container($color: #ccc);
}`

The color in `.mycard` will be equal to `$card-font-color` not to `#ccc`

I'm stating the the problem is that you set `$card-font-color` as default for `$color` parameter into the mixin, but then inside the mixin itself you totally ignore the $color parameter and apply `$card-font-color` to the color setting without looking at whatever the user passed in.

Compare the original with how `$background` and `$card-background` is handled.

How this currently works is equivalent to something like this

`@mixin headline($size, $color: red) {
  color: red; //No matter what color parameter a user set, the color will always be red
  font-size: $size;
}`